packages/dds/tree/src/test/feature-libraries/deltaUtils.spec.ts

```diff
@@ -35,49 +35,30 @@ describe("DeltaUtils", () => {
 				[
 					fooField,
 					[
-						{
-							type: Delta.MarkType.Modify,
```

Comment:
Why is this case being removed?

Reply:
Same for the other modify case removed below.

Reply:
I just decided that the likelihood of it not being handled properly despite the other Modify case to being handled properly was slim enough that a single case is fine.

By "the other modify case", do you mean the one that was on line 102? That the expected output of the `mapFieldMarks` function being tested, so deleting one requires deleting the other.